content/fast/reduce-network-payloads-using-text-compression/codelab-text-compression-brotli.md

```diff
@@ -176,15 +176,7 @@ For example, `main.js` would be renamed to `main.js.br`.
 
 When the app reloads and rebuilds, a compressed version of the main bundle is now created. Open the Glitch Console to take a look at what's inside the final `public/` directory that's served by the Node server.
 
-+ Click the **Status** button.
-
-<web-screenshot type="status"></web-screenshot>
-
-+ Click the **Console** button.
-
-<web-screenshot type="console"></web-screenshot>
-
-+ In the console, run the following commands to change into the `public` directory
+In the console, run the following commands to change into the `public` directory
```

Comment:
Sorry I meant just get rid of the web-screenshot elements.

You'll still want the instructions for opening the console. Here's an example: https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-codelab#instructions

Reply:
Specifically something like:

```
1. Click the **Tools** button.
1. Click the **Console** button.
```